scripts/generate_site_sidebar/generate_site_sidebar.go

```diff
@@ -0,0 +1,202 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+
+	"github.com/igorsobreira/titlecase"
+)
+
+const markdownExtension = ".md"
+const introPage = "00.md"
+
+func main() {
+	fmt.Print(`<!--This sidebar was automatically generated by 'make site-generate'-->
+- [Installation](installation/)
```

Comment:
Adding/removing content should not require compiling a Go program.

You can put this in a md file and use Go templating to process it.

Reply:
Switched to Go templating.